plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/helpers.test.ts

```diff
@@ -50,7 +51,7 @@ describe('fetchContent helper', () => {
     outputPath: os.tmpdir(),
   };
 
-  it('should reject absolute file locations', async () => {
+  it('fetch contents should reject absolute file locations', async () => {
```

Comment:
I'd wrap all these up in a separate `describe` block instead. as `it fetch contents` doesn't really read very well.

```ts
describe('fetch contents', () => {
  it('rejects absolute file locations')
})
```

Reply:
It looks much nicer, thanks for the suggestion.